interpreter/valid/valid.ml

```diff
@@ -402,10 +407,31 @@ let rec check_instr (c : context) (e : instr) (s : infer_stack_type) : op_type =
     let t1, t2 = type_cvtop e.at cvtop in
     [NumType t1] --> [NumType t2]
 
-  | TryCatch _ -> [] --> [] (* TODO *)
-  | TryDelegate _ -> [] --> [] (* TODO *)
-  | Throw _ -> [] --> [] (* TODO *)
-  | Rethrow _ -> [] --> [] (* TODO *)
+  | TryCatch (bt, es, cts, ca) ->
```

Comment:
To be consistent with `exec/eval.ml` I could move these to just below the other control instructions too.

Reply:
Fixed.